src/Relation/Binary/PropositionalEquality/Bundles.agda

```diff
@@ -0,0 +1,68 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Propositional (intensional) equality:
+--   Equivalence, Order, Setoid
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Relation.Binary.PropositionalEquality.Bundles where
```

Comment:
I'm not sure `Bundles` is quite the right word as we also have structures in here. If we were being consistent then we'd either put it in `Relation.Binary.PropositionalEquality.Properties` or call this `Relation.Binary.PropositionalEquality.Relation.Binary` 😆

I assume the former is problematic?

Reply:
I struggled with naming this one. The double `Relation.Binary` does seem to be a bit much. I could rename the current `Properties` to `Rearrange` and then `Bundles` could be `Properties`.

Reply:
You definitely need them to be separate? Up until now we've never distinguished between these two types of properties...

Reply:
Definitely? No. I split them mostly because they depend on somewhat different imports.

Reply:
Hmm there's only one import difference. If you don't need them to be different, then I'd be tempted to combine them and save us the trouble of coming up with a new naming convention?

Reply:
Ok, I'm probably going overboard there. I'll merge them.